weblate_web/templates/hosting.html

```diff
@@ -652,15 +652,14 @@ <h2 class="section-title crypto payment-conditions">{% trans "Payment" %}</h2>
             </div>
             <h2 class="section-title crypto payment-conditions" id="strings">{% trans "Source strings" %}</h2>
             <div class="page-desc small">
-                <p>{% trans "Pricing is based on number of source strings and target languages. The source string is a text unit defined in a translation format, it can be a word, sentence or paragraph." %}</p>
+                <p>{% trans "Pricing is based on the number of source strings and target languages. A source string is a text unit defined in a translation format, it can be a word, sentence or paragraph." %}</p>
```

Comment:
```suggestion
                <p>{% trans "Pricing is based on the number of source strings and target languages. A source string is a text unit defined in a translation format. It can be a word, sentence or paragraph." %}</p>
```